src/quo2/components/messages/system_message.cljs

```diff
@@ -151,7 +152,8 @@
   [system-message-base
    {:icon      {:icon    :i/pin
                 :color   (or customization-color :primary)
-                :opacity 5}
+                :opacity 5
+                :size    20}
```

Comment:
should it be medium? and probably it should be by default, so we don't need to specify it